friendshipper/src-tauri/src/server.rs

```diff
@@ -169,24 +167,7 @@ impl Server {
 
             let app = crate::router(&shared_state.log_path)?
                 .with_state(shared_state.clone())
-                .layer(axum::middleware::from_fn(uri_passthrough))
```

Comment:
Why did the axum layer get removed?

Reply:
we get this behavior for free from the new layer (the uri is recorded in every span)